.github/workflows/npm-test.yml

```diff
@@ -76,3 +76,26 @@ jobs:
       # Check that the storybook build succeeds
       - name: Build Storybook
         run: npm run build:storybook
+
+      # Install Playwright (with cache) for Storybook tests
+      - name: Get installed Playwright version
+        id: playwright-version
+        run: echo "PLAYWRIGHT_VERSION=$(node -e "console.log(require('./package-lock.json').dependencies['@playwright/test'].version)")" >> $GITHUB_ENV
+      - name: Cache playwright binaries
+        uses: actions/cache@v4
+        id: playwright-cache
+        with:
+          path: |
+            ~/.cache/ms-playwright
+          key: ${{ runner.os }}-playwright-${{ env.PLAYWRIGHT_VERSION }}
+      - run: npx playwright install --with-deps
+        if: steps.playwright-cache.outputs.cache-hit != 'true'
+      - run: npx playwright install-deps
+        if: steps.playwright-cache.outputs.cache-hit != 'true'
+
+      # Run the storybook test suite
+      - name: Serve Storybook and run tests
```

Comment:
add this to `ci_locally.sh` as well

Reply:
I see that `ci_locally.sh` is running `npm run ci`, which is running `npm run test`, which now has storybooks tests in it :)